nss-day-cohort-60/national-parks-django-api
# clients can now view list of amenities, individual amenities, and amenities by park

Description:

Added get all, get by id, and get by park id to amenty_view.py

Steps to test:

Fetch the md-amenties-view branch and switch to it
Start/restart debugger/server
Open Postman and use to make the following requests:

1. Send a GET request for all amenities (example: http://localhost:8000/amenities)
   Verify that 200 status code and that all amenities are displayed with id and type
2. Send a GET request for a single amenity (example: http://localhost:8000/amenities/2)
Verify that 200 status code and that the amenity with the id of 2 is displayed only
3. Send a GET request for all amenities by park_id (example: http://localhost:8000/amenities?park_id=2)
Verify that 200 status code and that all amenities from that particular park are displayed
